src/components/medications-details-table.component.tsx

```diff
@@ -0,0 +1,284 @@
+import React, { useState } from 'react';
+import {
+  Button,
+  DataTable,
+  OverflowMenu,
+  OverflowMenuItem,
+  Pagination,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableHeader,
+  TableRow,
+  TableToolbar,
+  TableToolbarContent,
+} from 'carbon-components-react';
+import { getDosage } from '../utils/get-dosage';
+import dayjs from 'dayjs';
+import { useTranslation } from 'react-i18next';
+import { Add16 } from '@carbon/icons-react';
+import { capitalize } from 'lodash-es';
```

Comment:
Can we check if this gets bundled efficiently? Potentially, `lodash-es/capitalize` (or similar) would be more efficient.